src/main/webapp/js/userMapData.json

```diff
@@ -132,16 +142,20 @@
 ["City College","Greece"],
 ["MIO-ECSDE","Greece"],
 ["Colegio Americano del Sur","Guatemala"],
+["CCC Rotary Secondary School","Hong Kong"],
 ["HKU SPACE Community College","Hong Kong"],
 ["Queen Elizabeth School","Hong Kong"],
 ["The Hong Kong Polytechnic University","Hong Kong"],
 ["University of Hong Kong","Hong Kong"],
+["Codecool","Hungary"],
 ["All India Institute of Medical Sciences","India"],
+["Bharati Vidypeeth College of Engineering, New Delhi","India"],
```

Comment:
actually @chowyb wait. 

@damithc should we edit this? I think you have generated the list based on the institute names given by the instructors themselves right? Maybe we should leave it as it is....